stubs/docker/docker/types/containers.pyi

```diff
@@ -172,3 +172,8 @@ class ContainerConfig(dict[str, Incomplete]):
         stop_timeout: int | None = None,
         runtime: str | None = None,
     ) -> None: ...
+
+@type_check_only
+class TopResult(TypedDict):
```

Comment:
Typecheck-only types need to be prefixed with `_` to make it clear to users that this is "special" in some way, even if their IDE or editor doesn't point out the fact that it can't be imported at runtime.